descartes_planner/src/planning_graph.cpp

```diff
@@ -113,26 +112,25 @@ bool PlanningGraph::insertGraph(const std::vector<TrajectoryPtPtr> *points)
   }
 
   // after populating maps above (presumably from cartesian trajectory points), calculate (or query) for all joint trajectories
-  if (!calculateJointSolutions())
+  std::vector<std::vector<JointTrajectoryPt>> poses;
+  if (!calculateJointSolutions(*points, poses))
   {
     // failed to get joint trajectories
     ROS_ERROR_STREAM("unable to calculate joint trajectories for input points");
     return false;
   }
 
-  if (!populateGraphVertices())
+  std::vector<JointEdge> edges;
+  if (!calculateAllEdgeWeights(poses, edges))
   {
-    ROS_ERROR_STREAM("unable to populate graph from input points");
+    // failed to get edge weights
```

Comment:
Having a commend and an error message that describe about the same feels redundant.